samples/azure/azure-service-bus-msmq-bridge/ASB_7/Bridge/Program.cs

```diff
@@ -24,23 +24,32 @@ static async Task Main(string[] args)
                 //Prevents ASB from using TransactionScope
                 transport.Transactions(TransportTransactionMode.ReceiveOnly);
                 transport.ConnectionString(connectionString);
-                transport.UseForwardingTopology();
+                var topology = transport.UseEndpointOrientedTopology();
+                topology.RegisterPublisher(typeof(OtherEvent), "Samples.Azure.ServiceBus.AsbEndpoint");
             });
 
         bridgeConfiguration.AutoCreateQueues();
         bridgeConfiguration.UseSubscriptionPersistece<InMemoryPersistence>((configuration, persistence) => { });
+        bridgeConfiguration.TypeGenerator.RegisterKnownType(typeof(OtherEvent));
 
         #endregion
 
+        #region resubscriber
+        var resubscriber = await Resubscriber<MsmqTransport>.Create("Bridge-MSMQ", TimeSpan.FromSeconds(10), t => { });
```

Comment:
> TimeSpan.FromSeconds(10), t => { }

I suspect it's a refresh interval and refresh action?
Adding parameter names would make it clear.